### PR TITLE
First README draft

### PR DESCRIPTION
**Pull request by AntoineAugusti**
Related to #7

### CODE REVIEW

**Comment by AntoineAugusti** [comment]
Hey @bzg and @jbledevehat,

Can you read the README and give me your feedback? Stick to what you know and understand.

If you think that I should explain a bit more some sections or that I should add more content, please say it.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Contributing
+
+Contributions are **welcome** and will be fully **credited**. We accept contributions via pull requests on GitHub.
```
Comment (bzg) [inline CONTRIBUTING.md]: No need to credit every contributor. I'd suggest this:

> Contributions are **welcome**. Most active contributors will be credited in the `README.md`. We accept contributions via pull requests on GitHub.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## Pull Requests
+
+- **[PSR-2 Coding Standard.](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md)** The easiest way to apply the conventions is to install [PHP CS Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer).
+- **Ask before working on a feature.** Before spending hours on a feature, open an issue and let's discuss about it.
+- **Add tests!** Your patch won't be accepted if it doesn't have tests.
```
Comment (bzg) [inline CONTRIBUTING.md]: > **Add tests!** Your patch won't be accepted if it doesn't have tests.

I suggest:

> **Add tests!** Patches get accepted only when they include tests.

This leaves the possibility to submit a PR without tests and work on adding tests while discussing the PR.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+- **Document any change in behaviour.** Make sure the `README.md` and any other relevant documentation are kept up-to-date.
+- **Create feature branches.** Don't ask us to pull from your master branch.
```
Comment (bzg) [inline CONTRIBUTING.md]: Don't ask us to pull from your master branch.

The tone is not very welcoming. I suggest:

> Small fixes can be pulled from your master branch, but features needs to be pulled from a feature branch.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+[![Software License](https://img.shields.io/badge/License-AGPL-orange.svg?style=flat-square)](https://github.com/entrepreneur-interet-general/ma-semaine/blob/master/LICENSE)
+
+# What is it
+$APP_NAME is a weekly retrospective tool for multiple projects or teams. It lets people reflect on their past week with 4 questions which can be answered super quickly:
+- What's the team spirit?
+- What were the main goals this week?
+- What worked great and what was harder?
+- Do we need help?
+
+It's asynchronous and transparent at its heart. All teams can fill their retrospective when they want through a simple web interface, as long as it's before Friday 3 PM. On Fridays at 3 PM, everyone gets a weekly recap email with all filled retrospectives. The web interface lets everyone browse through previous retrospectives by week or by team.
```
[the remark this answers is not in the document]
Comment (AntoineAugusti) [reply README.md]: Yup it's hardcoded for now

File: README.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+php artisan migrate
+```
+
+#### App settings
+Regarding `.env` specific to the app, you should file the following variables:
+
+- `REPORT_TIMEZONE`: The timezone to use for reports. Use a time zone name from the IANA database like Europe/Paris.
+- `REPORT_EMAIL`: To which email address should weekly reports be sent to. Example: `team@company.com`
+- `REPORT_SECRET`: The password to see previous reports in the web interface.
+- `REPORTS_PASSWORD_HINT`: An hint to guess the password of the web interface. If you don't want to provide an hint, leave it null.
```
[the remark this answers is not in the document]
Comment (AntoineAugusti) [reply README.md]: Because with a single password you can see all reports

File: README.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+- We don't provide an authentication system. It should be deployed internally or can be secured with something like HTTP Auth Basic.
+- Transparency is important. When teams fill their retrospective, it can be read by all other teams. Everyone who has access to the application can browse through previous retrospectives and export them in CSV.
+- Teams are free to fill their weekly retrospective when they want between Sunday midnight and Friday 3PM.
+- Teams are reminded to fill their weekly retrospective in various ways:
+    - a collective Slack reminder on a channel you can define on Fridays at 10 AM
+    - by DM on Slack to each member of a team if the retrospective of the team has not be filled yet on Fridays at 2 PM and 2:45 PM
+- At 3 PM every Friday, all filled weekly retrospective are shared by email. This should typically be a mailing list where everyone is subscribed to.
+- Previous retrospectives can be seen on the web interface after authentication with a password you can define. You can define a password hint to help people remember the password.
```
[the remark this answers is not in the document]
Comment (AntoineAugusti) [reply README.md]: There is no user model. Just a simple password authentication when looking at previous reports.

File: README.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
+[![CircleCI build status](https://img.shields.io/circleci/project/github/entrepreneur-interet-general/ma-semaine.svg?style=flat-square)](https://circleci.com/gh/entrepreneur-interet-general/ma-semaine)
+[![Software License](https://img.shields.io/badge/License-AGPL-orange.svg?style=flat-square)](https://github.com/entrepreneur-interet-general/ma-semaine/blob/master/LICENSE)
+
+# What is it
+$APP_NAME is a weekly retrospective tool for multiple projects or teams. It lets people reflect on their past week with 4 questions which can be answered super quickly:
```
[the remark this answers is not in the document]
Comment (AntoineAugusti) [reply README.md]: Not convinced yet

**Comment by AntoineAugusti** [comment]
@bzg thanks a lot for your useful comments! I used a template for the CONTRIBUTING file, hence the tone sometimes. I tackled a large majority of your comments

**Comment by AntoineAugusti** [comment]
I'm merging as is for now. Feel free to suggest updates by opening issues!

**Pull request merged**